spython/instance/cmd/stop.py

```diff
@@ -29,6 +31,9 @@ def stop(self, name=None, sudo=False):
     if "version 3" in self.version():
         subgroup = ["instance", "stop"]
 
+    if timeout:
```

Comment:
okay, so if Singularity version 2 ran this command the user would get an error from Singularity that there is no timeout, do you think that's okay?

Reply:
Sorry, I had already commited before receiving your previous comment, this way should be ok